website/content/docs/ui/custom-login.mdx

```diff
@@ -0,0 +1,543 @@
+---
+layout: docs
+page_title: Manage custom login options
+description: >-
+  Use custom login flows to control how clients can authenticate to Vault
+  through the GUI.
+---
+
+# Manage custom login options for the Vault GUI
+
+@include 'alerts/enterprise-and-hcp.mdx'
+
+Use login customization to set preferred authentication methods and control which methods display in the Vault GUI login form.
+
+## Before you start 
+
+@include 'alerts/restricted-admin.mdx'
+
+- **You must have Vault Enterprise 1.20.0 or higher installed.**
+- **You must have the appropriate permissions**:
+  - You must have `update` permission for the [`sys/config/ui/login/default-auth`](/vault/api-docs/system/config-ui-login-default-auth) endpoint.
+ 
+## Step 1: Select your authentication method
+
+Only the following methods are supported by the GUI and can be set as a default or backup method:
+
+@include 'ui/supported-login-methods.mdx'
+
+You can customize your login flow with a default authentication method,
+backup authentication methods, or both.
+
+Configuration           | Result
+----------------------- | ------
+Default only            | User only see the selected authentication method during login.
+Backup(s) only          | User only sees from the selected authentication method(s) during login.
+Default and backup(s)   | User see the default method first but can access the backup methods as alternate options.
+
+
+## Step 2: Make the authentication method visible to the GUI. (Optional, but recommended) 
+
+Set the [listing visibility](/vault/api-docs/system/auth#listing_visibility-1) of your authentication plugin to `unauth` 
+so the Vault GUI knows which login URL to authenticate against.
+
+<Warning title="Listing visibility and plugin mount paths">
+
+Login customizations specify authentication **types**, not specific plugin paths.
+As a result, the Vault GUI assumes each login method is mounted at its default mount [path](/vault/api-docs/system/auth#path), 
+which corresponds to its auth type.
```

Comment:
Really struggling with wording here - this is a pretty important piece of information to get across